distribution/ddf-common/src/main/resources/security/configurations.policy

```diff
@@ -1,80 +1,133 @@
-// equivalent.
+// * CHANGES TO THIS FILE REQUIRE THE SYSTEM TO BE RESTARTED
+//   This file is read when the system is started. Changes to this file are only picked up when
+//   the system is started.
 //
-// IMPORTANT! - Adding new permissions requires a system restart to take affect.
-
-//                        Content Directory Monitor (CDM) Permissions
-// Adding required permissions for a new Content Directory Monitor.
+// * PLATFORM INDEPENDENT FILE PATHS
+//   Use the special notation of "${/}", which is a shortcut for "${file.separator}".
+//   It inserts the property separator for the current platform (Windows or *nix).
+//
+// * WINDOWS DRIVE LETTERS
+//   Window drive letters are specified at the beginning of the file path.
+//   For example, the path:
+//       "D:\my\files"
+//   would be entered as
+//       "D:${/}my${/}files"
 //
-// If monitoring a WebDav server, then adding these permissions is not required and these instructions can be ignored.
 //
-// Configuring CDM requires adding read and write permissions to the directory being monitored. For
-// each configured CDM, the following 2 permissions are required:
-//   1. permission java.io.FilePermission "<DIRECTORY_PATH>", "read";
-//   2. permission java.io.FilePermission "<DIRECTORY_PATH>${/}-", "read, write";
-// Replacing <DIRECTORY_PATH> with the path of the directory being monitored.
+// * TRAILING FILE SEPARATORS ARE IGNORED
+//   The paths "${/}test" and "${/}test${/}" are equivalent
 //
-// Line 1 gives the CDM the permissions to read from the monitored directory path. Line 2 gives the
-// CDM the permissions to recursively read and write from the monitored directory path, specified
-// by the directory path's suffix "${/}-".
+// * GRANTING PERMISSIONS RECURSIVELY
+//   A file path with a trailing "${/}-" (a slash and a dash) matches all files in the directory
+//   and recursively all files in subdirectories contained in that directory.
 //
-// If a CDM configuration is deleted, then the corresponding permissions that were added should be
-// deleted to avoid granting unnecessary permissions to parts of the system.
+// * COMMENTS
+//   Lines that begin with double forward slashes "//" indicate comments
+//   and have no effect on policies.
 //
+// * DOUBLE QUOTES
+//   File paths and URLs should be surrounded in double quotes.
+//
+// *************************************************************************************************
+
+
+
+// ***********************  BEGIN CONTENT DIRECTORY MONITOR SECTION  *******************************
+// * Configuring the CDM requires adding read and write permissions to the directory being monitored.
+// * If monitoring a WebDav server, then adding these permissions is not required and
+//   these instructions can be ignored.
+// * If a CDM configuration is deleted, then the corresponding permissions that were added should be
+//   removed from this file to avoid granting unnecessary permissions to parts of the system.
```

Comment:
✏️ Might be a good idea to add a note here that the URL resource reader also needs permissions to read from the directory if the CDM is in-place.